src/router.js

```diff
@@ -25,7 +26,27 @@ export default new Router({
     {
       path: '/books',
       name: 'books',
-      component: () => import('./views/Books.vue'),
+      component: () =>
+        import(/* webpackChunkName: "books" */ './views/Books.vue'),
+    },
+    {
+      path: '/users',
+      name: 'users',
+      component: () =>
+        import(/* webpackChunkName: "users" */ './views/Users.vue'),
+      beforeEnter: (to, from, next) => {
+        if (!store.getters.loggedIn) {
+          next(false);
```

Comment:
I have one question regarding this (I don't remember how it works). What does it do if you are not logged in? It navigates to special page?

Reply:
It's triggered when you go to that page, so it won't do anything. Right now you won't even get feedback. On another task, I want to add error messages..

Also, if you open a new browser or tab and try to go directly, it goes to the default page